repository.eselect.in

```diff
@@ -405,6 +417,12 @@ do_list() {
 					name+=" $(highlight_warning '#')"
 				fi
 				;;
+			*)
+				if [[ -n ${only_installed} ]]; then
+					i=$(( i + 1 ))
```

Comment:
Maybe:

    (( i++ ))

for consistency.

Reply:
I couldn't get this to work but will retry

Reply:
This does not work.. "repository.eselect: line 423: 400: command not found" if by itself and it counts incorrectly if i=